src/Libraries/Microsoft.Extensions.Http.Resilience/Microsoft.Extensions.Http.Resilience.csproj

```diff
@@ -25,6 +25,12 @@
     <MinMutationScore>100</MinMutationScore>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Disable "Published symbols cannot be deleted to maintain compatibility" warning
+         because we have different APIs for different TFMs. -->
+    <NoWarn Condition="'$(TargetFramework)' == 'net462'">$(NoWarn);LA0006</NoWarn>
+  </PropertyGroup>
```

Comment:
We have to suppress `LA0006 - Published symbols cannot be deleted to maintain compatibility` warning. The reason is the following. Together with making the `ResilienceHandler` class stable, we also made stable protected members `SendAsync` and `Send`. These members are inherited from `DelegatingHandler` and `ResilienceHandler` overrides them. The `Send` method is not available on `net462` and this causes the build pipeline to fail with the `LA0006` warning. To make the build pipeline "green" we have to suppress the `LA0006` warning.

Reply:
This is suppressing all of the warnings on public symbols though, right? Is there not a way to suppress that single change?

Reply:
Also, I'm not very familiar to which analyzer is throwing those errors, but FWIW we do run package validation when we pack projects, which would catch compatibility issues in packages. In this case, net462 and net6.0+ aren't even supposed to be compatible with each other so perhaps the analyzer tooling being used here is not correct.

Reply:
@joperezr 

> This is suppressing all of the warnings on public symbols though, right? Is there not a way to suppress that single change?

Sometime ago I tried to suppress the warning in a source file for a particular change, but it didn't work. Here is another fresher PR https://github.com/dotnet/extensions/pull/5690 where I tried it again and it didn't work. Unfortunately, we have to suppress the warning for the whole project.

> Also, I'm not very familiar to which analyzer is throwing those errors, but FWIW we do run package validation when we pack projects, which would catch compatibility issues in packages. In this case, net462 and net6.0+ aren't even supposed to be compatible with each other so perhaps the analyzer tooling being used here is not correct.

Yes, I agree. I don't understand what exactly the analyzer is catching. Adding a non-abstract method that is available in net6+ and is not on net462 is alright in regards of compatibility, i.e. upgrading from net462 to net6+ will not break. Right?

Reply:
@joperezr I'm going to resolve this discussion and merge the PR. Do you know who we can follow up with on this analyzer warning? We need to figure out whether the warning the analyzer triggers is incorrect.

Reply:
@iliar-turdushev please open a new issue and let's continue the discussion there.

Reply:
@RussKie Here is the issue https://github.com/dotnet/extensions/issues/5721.